features/step_definitions/back_office/edit_registration_steps.rb

```diff
@@ -13,7 +13,20 @@
 
 end
 
-Then(/^I will be informed the registration is revoked$/) do
+When(/^I cease a registration$/) do
+  @app.search_page.menu_home.click
+  @app.search_page.search_field.set @exemption_number
+  @app.search_page.search_button.click
+  @app.search_page.first_search_result.click
+
+  @app.registration_details_page.deregister.click
+
+  @app.deregistration_page.cease(
```

Comment:
In line with my previous comment about following the `submit` page pattern, this would change to

```ruby
@app.derehistration_page.submit(
  reason: :cease,
  deregistration_comment: "This is a test"
)
```